dataprofiler/profilers/profile_builder.py

```diff
@@ -153,18 +155,37 @@ def __add__(self, other):
             options=self.options,
         )
 
+        # Check if indices overlap, if they do, adjust attributes accordingly
+        other_min_id = other.min_id
+        other_max_id = other.max_id
+        other_nti = copy.deepcopy(other.null_types_index)
+        if utils.overlap(self.min_id, self.max_id, other.min_id, other.max_id):
+            warnings.warn(f"Attempted to merge profiles with overlapping "
+                          f"indices. The indices in {other.name} will be "
+                          f"shifted in the merged profile to resolve this.")
+
+            other_min_id = other.min_id + self.max_id + 1
+            other_max_id = other.max_id + self.max_id + 1
+
+            other_nti = {k: utils.increment(v, self.max_id + 1) for k, v in
+                         other.null_types_index.items()}
```

Comment:
double plus one?

Reply:
The top 2 lines are incrementing the other.min_id and other.max_id, the bottom line increments every row index in other.null_type_index, so there is no double since these are for different attributes